db/migrate/0_enable_extensions.rb

```diff
@@ -0,0 +1,6 @@
+class EnableExtensions < ActiveRecord::Migration[5.0]
+  def change
+    enable_extension 'uuid-ossp'
+    enable_extension 'pgcrypto'
+  end
+end
```

Comment:
I think I had this problem before when I added a new model, but I think I somehow used the old style for ID's and got around it at the time because I wasn't sure what the problem was.